includes/collection/class-remote-actors.php

```diff
@@ -451,6 +451,20 @@ public static function get_actor( $post ) {
 			self::add_error( $post->ID, $actor );
 		}
 
+		if ( ! $actor->get_webfinger() ) {
+			$webfinger = \get_post_meta( $post->ID, '_activitypub_acct', true );
+
+			if ( ! $webfinger ) {
+				$acct = Webfinger::uri_to_acct( $actor->get_id() );
+				if ( \is_wp_error( $acct ) ) {
+					$acct = Webfinger::guess( $actor );
+				}
+				$webfinger = Sanitize::webfinger( $acct );
+			}
+
+			$actor->set_webfinger( $webfinger );
```

Comment:
```suggestion
			$actor->set_webfinger( Webfinger::get_acct( $post->ID ) );
```

`Webfinger::get_acct()` also saves it to post meta so it would only have to be fetched once.

Reply:
We can not do that, otherwise we have an infinite loop, because `get_acct` is calling `get_actor`.